proxy-robots-module/src/main/java/orwell/proxy/robot/IPWebcam.java

```diff
@@ -13,12 +13,12 @@
     private final URL url;
 
     public IPWebcam(final URL url) {
-        assert(null != url);
+        assert (null != url);
```

Comment:
I am quite used to consider assert a normal function. Why did you add a space ? Is it a Java style thing ?